app/src/main/kotlin/com/flxrs/dankchat/chat/ChatAdapter.kt

```diff
@@ -76,7 +76,7 @@ class ChatAdapter(
         with(holder.binding.itemText) {
             when (val message = getItem(position).message) {
                 is Message.SystemMessage -> handleSystemMessage(message)
```

Comment:
Maybe add the checkered background also to system messages?

Reply:
see c0dafb0eb167c0c30cf791651487e70cf1c82f4c

Reply:
Do you have any tests for that ? I can't seem to "fake" a system message.

Reply:
The initial "Connected" message is a system message for example, tested it and it works 👍